core/src/main/scala/ai/h2o/sparkling/backend/api/dataframes/DataFramesRestApi.scala

```diff
@@ -0,0 +1,45 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package ai.h2o.sparkling.backend.api.dataframes
+
+import java.net.URI
+
+import ai.h2o.sparkling.backend.utils.RestApiUtils
+import org.apache.spark.h2o.H2OContext
+
+trait DataFramesRestApi extends RestApiUtils {
+
+  protected def listDataFrames(): DataFrames = {
+    val hc = H2OContext.ensure()
+    val endpoint = new URI(hc.flowURL())
+    query[DataFrames](endpoint, "/3/dataframes", hc.getConf)
```

Comment:
nit: Doesn't existing path start with a capital letter `/3/Dataframes`? I'm not sure, just asking..

Reply:
Good questions! It is lowercase ( inconsistent, but would like the endpoints to remain the same as part of this change)